portfolio/src/main/webapp/script.js

```diff
@@ -34,15 +34,37 @@ async function getName() {
 }
 
 async function getComment() {
+  const visibility = document.getElementById("hideorshow");
+  const hidelogin = document.getElementById("hidelogin");
+  const showlogout = document.getElementById("showlogout");
+  const login = await fetch('/login');
+  const loginstatus = await login.text();
+  const loginName = loginstatus.split("<p");
+  if(loginName[0].includes("please sign in to leave a comment.")){
+      visibility.style.display = "none";
+      hidelogin.style.display = "block";
+      showlogout.style.display ="none";
+  }else{
+   showlogout.style.display= "block";
+   hidelogin.style.display= "none";
+   visibility.style.display = "block";
   const response = await fetch('/data');
   const comments = await response.json();
   const historyEl = document.getElementById('history');
-  for(var i = 0; i < comments.length; i++){
+  for(let i = 0; i < comments.length; i++){
     historyEl.appendChild(createListElement(comments[i]));
   }
-    
+  
+  }  
 }
 
+async function loginIn() {
+  const response = await fetch('/login');
+  const user = await response.text();
+  const username = user.split("<p");
```

Comment:
Instead of splitting the text tags, I think you would have an easier time if you returned JSON from the login servlet. Something like

```json
{
  "username": "testuser@example.com",
  "logoutUrl": "http://example.com/buhbye"
}
```